modules/claims_api/app/controllers/claims_api/v2/veterans/intent_to_file_controller.rb

```diff
@@ -71,6 +62,49 @@ def validate
 
         private
 
+        def intent_to_file_options(type)
+          options = {
+            intent_to_file_type_code: type,
+            participant_vet_id: params[:participant_vet_id] || target_veteran.participant_id,
```

Comment:
this would be camelCased according to the documentation, so `participant_vet_id` would not retrieve the value.

Reply:
Also, if the documentation is correct. I feel like we're wasting `participantVetId`. I think it's safe to assume that the `veteranId` (the ICN being passed in the URL) would act as the `participantVetId` so I believe we could remove that requirement for a request param. https://dev-developer.va.gov/explore/benefits/docs/claims?version=current